pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -93,6 +93,10 @@ const (
 	// ProxyInboundListenPort is the port on which all inbound traffic to the pod/vm will be captured to
 	// TODO: allow configuration through mesh config
 	ProxyInboundListenPort = 15006
+
+	// Used in xds config. Metavalue bind to this key is used by pilot as xds server but not by envoy.
```

Comment:
Can you explain in more details, I'm not sure I understood. An example ?

Reply:
This is used for EnvoyFilter matching... pilot level only.